### DISCUSSION TITLE
Speed up looking to see if point is over land or water.

### DISCUSSION THREAD

**Discussion by look4terry** [DotSpatial/DotSpatial#1498] [Q&A]
I'm working on an application written with .netCore 3.1.  I've found found .shp files released by NOAA that show coastlines.  I've written small that seems to work by iterating through the features in a .shp file and uses the contains method.  It is slow.  NOAA provides .shx files too.  How can I use the spatial indexes to speed things up.  Is there an example somewhere?

Thanks

**Comment by sindizzy** [comment]
What about using the Disjoint geometry function first?
